virtkvmqe/migration_test_plan/migration/test_scenarios/rhel7_59369.py

```diff
@@ -0,0 +1,117 @@
+    output = src_remote_qmp.qmp_cmd_output(cmd=cmd, recv_timeout=5)
+    if re.findall('error', output):
+        test.test_error('Failed to execute cpu hotplug command')
+    else:
+        test.test_print('Cpu hotpulg command is executed successfully')
+
+    test.main_step_log('4. Verify the core has been hot-plugged to the guest')
+    test.sub_step_log('4.1 Check cpu info inside guest')
+    cmd = "lscpu | sed -n '3p' | awk '{print $2}'"
+    output = src_serial.serial_cmd_output(cmd=cmd, recv_timeout=5)
+    if re.findall(r'4', output):
+        test.test_print('The guest cpus info checked inside guest is right')
+    else:
+        test.test_error('The guest cpus info checked inside guest is wrong')
+
+    test.sub_step_log('4.2 Check cpu info in src host')
+    cmd = '{"execute": "query-hotpluggable-cpus"}'
+    output = src_remote_qmp.qmp_cmd_output(cmd=cmd ,recv_timeout=5)
+    if re.findall(r'"qom-path": "/machine/peripheral/core1"', output):
+        src_remote_qmp.test_print('The guest cpus info checked in src host '
+                                  'is right')
+    else:
+        src_remote_qmp.test_print('The guest cpus info checked in src host '
+                                  'is wrong')
+
+    test.main_step_log('5. Boot the destination guest '
+                       '(with the hot-plugged core)')
+    cpu_val = 'host-spapr-cpu-core,core-id=2,id=core1'
+    params.vm_base_cmd_add('device', cpu_val)
+    incoming_val = 'tcp:0:%s' % incoming_port
+    params.vm_base_cmd_add('incoming', incoming_val)
+    dst_qemu_cmd = params.create_qemu_cmd()
+    src_host_session.boot_remote_guest(cmd=dst_qemu_cmd,
+                                       ip=DST_HOST_IP, vm_alias='dst')
+    dst_remote_qmp = RemoteQMPMonitor(id, params, DST_HOST_IP, qmp_port)
+    cmd = '{"execute": "query-hotpluggable-cpus"}'
+    output = dst_remote_qmp.qmp_cmd_output(cmd=cmd ,recv_timeout=5)
+    if re.findall(r'"qom-path": "/machine/peripheral/core1"', output):
+        src_remote_qmp.test_print('The guest cpus info of dst host is right')
+    else:
+        src_remote_qmp.test_print('The guest cpus info of dst host is wrong')
+
+    test.main_step_log('6. Migrate guest from source to destination and '
+                       'wait until it finishes,quit qemu of src host')
+    flag = do_migration(remote_qmp=src_remote_qmp,
+                        migrate_port=incoming_port, dst_ip=DST_HOST_IP)
+    if (flag == False):
+        test.test_error('Migration timeout')
+    output = src_remote_qmp.qmp_cmd_output('{"execute":"quit"}',
+                                           recv_timeout=5)
+    if output:
+        src_remote_qmp.test_error('Failed to quit qemu on src host')
+
+    test.main_step_log('7. Run some command onto the hot-plugged core')
+    dst_serial = RemoteSerialMonitor(id, params, DST_HOST_IP, serial_port)
+    time.sleep(10)
+    cmd = 'taskset -c 2,3 dd if=/dev/urandom of=/dev/null bs=64K count=1000'
```

Comment:
I think that command dd should be added a option "oflag=direct" which flush the data to disk.